src/safeds/ml/classical/classification/_baseline_classifier.py

```diff
@@ -0,0 +1,185 @@
+        The original model is not modified.
+
+        Parameters
+        ----------
+        train_data:
+            The data the network should be trained on.
+
+        Returns
+        -------
+        trained_classifier:
+            The trained Classifier
+
+        Raises
+        ------
+        DatasetMissesDataError
+            If the given train_data contains no data.
+        ColumnTypeError
+            If one or more columns contain non-numeric values.
+        """
+        from concurrent.futures import ProcessPoolExecutor
+
+        # Validate Data
+        train_data_as_table = train_data.to_table()
+        if train_data_as_table.row_count == 0:
+            raise DatasetMissesDataError
+        _check_columns_are_numeric(train_data_as_table, train_data.features.add_columns(train_data.target).column_names)
+
+        copied_model = copy.deepcopy(self)
+
+        with ProcessPoolExecutor(max_workers=len(self._list_of_model_types)) as executor:
+            futures = []
+            for model in self._list_of_model_types:
+                futures.append(executor.submit(_fit_single_model, model, train_data))
+            [done, _] = wait(futures, return_when=ALL_COMPLETED)
+            for future in done:
+                copied_model._fitted_models.append(future.result())
+        executor.shutdown()
+
+        copied_model._is_fitted = True
+        copied_model._feature_names = train_data.features.column_names
+        copied_model._target_name = train_data.target.name
+        return copied_model
+
+    def predict(self, test_data: TabularDataset) -> dict[str, float]:
+        """
+        Make a prediction for the given test data and calculate the best metrics.
+
+        The original Model is not modified.
+
+        Parameters
+        ----------
+        test_data:
+            The data the Classifier should predict.
+
+        Returns
+        -------
+        best_metrics:
+            A dictionary with the best metrics that were achieved.
+
+        Raises
+        ------
+        ModelNotFittedError
+            If the model has not been fitted yet
+        FeatureDataMismatchError
+            If the features of the test data do not match with the features of the trained Classifier.
+        DatasetMissesDataError
+            If the given test_data contains no data.
+        DatasetMissesTargetError
+            If the given test_data misses the target column.
+        ColumnTypeError
+            If one or more columns contain non-numeric values.
+        """
+        from concurrent.futures import ProcessPoolExecutor
+
+        from safeds.ml.metrics import ClassificationMetrics
+
+        if not self._is_fitted:
+            raise ModelNotFittedError
+
+        # Validate data
+        if not self._feature_names == test_data.features.column_names:
+            raise FeatureDataMismatchError
+        if not self._target_name == test_data.target.name:
+            raise DatasetMissesTargetError(self._target_name)
```

Comment:
maybe rename this error in DatasetTargetDoesNotMatch or something in that direction
The name suggest, that a target column is missing, but they are not matching